packages/notte-browser/src/notte_browser/session.py

```
@@ -27,11 +28,13 @@
 from notte_sdk.types import (
     DEFAULT_MAX_NB_STEPS,
     BrowserType,
+    Cookie,
     PaginationParams,
     PaginationParamsDict,
     ProxySettings,
     ScrapeParams,
     ScrapeParamsDict,
+    SetCookiesRequest,
```

Ruff (0.8.2)

`notte_sdk.types.SetCookiesRequest` imported but unused

Remove unused import: `notte_sdk.types.SetCookiesRequest`

(F401)